cli/internal/runsummary/execution_summary.go

```diff
@@ -107,10 +110,10 @@ type executionSummary struct {
 	profileFilename string
 
 	// These get serialized to JSON
-	success   int
-	failure   int
-	cached    int
-	attempted int
+	success   int // number of tasks that exited successfully (does not include cache hits)
+	failure   int // number of tasks that exited with failure
+	cached    int // number of tasks that had a cache hit
+	attempted int // number of tasks that started
```

Comment:
This is still a little confusing. I would expect:

success + failure = attempted. And cached is <= success

Reply:
yeah agreed, but I don't want to break how `success` is used right now